arbos/l1pricing/l1pricing.go

```diff
@@ -63,6 +64,7 @@ const (
 	perUnitRewardOffset
 	lastUpdateTimeOffset
 	fundsDueForRewardsOffset
+	brotliCompressionLevelOffset
```

Comment:
New entries should be added at end of list, otherwise will change storage offset for everything listed after.